doc/source/whatsnew/v0.24.0.txt

```diff
@@ -159,6 +159,56 @@ This is the same behavior as ``Series.values`` for categorical data. See
 :ref:`whatsnew_0240.api_breaking.interval_values` for more.
 
 
+.. _whatsnew_0240.enhancements.duplicated_inverse:
+
+The ``duplicated``-method has gained the ``return_inverse`` kwarg
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ``duplicated``-method for ``Series``, ``DataFrame`` and all flavors of ``Index`` has gained a ``return_inverse`` keyword,
```

Comment:
Double backticks are for literals and code samples, so let's put `False` in double backticks and everything else that you have here rendered with just single backticks, save `return_inverse=True` which is OK

Reply:
I see I commented now again the reverse .. 
But @WillAyd, I think you gave the wrong recommendation, the "Double backticks are for literals and code samples" is correct, but we see method names as "code samples" (although for duplicated it is actually better to use the link-markup `` :meth:`~DataFrame.duplicated` ``). 
In general I don't think there should be much single backticks in normal .rst files (except for those sphinx mark-up cases like refs)

In docstrings we *do* use single backticks to refer to parameter names.